tests/rt.sh

```diff
@@ -265,15 +265,15 @@ elif [[ $MACHINE_ID = hercules ]]; then
   export PATH=/apps/spack-managed/gcc-11.3.1/miniconda3-4.10.3-un3f2xdus7rbrzgso5ketsq4gp2iociv/bin:$PATH
   export PYTHONPATH=/apps/spack-managed/gcc-11.3.1/miniconda3-4.10.3-un3f2xdus7rbrzgso5ketsq4gp2iociv/lib/python3.9/site-packages
 
-  module use /work/noaa/epic-ps/role-epic-ps/spack-stack/modulefiles
+  module use /work/noaa/epic/role-epic/spack-stack/modulefiles
```

Comment:
this modulefiles dir. and the packages that the modules will point to have yet to be installed on hercules in this new space (reiterating my comment from the pr conversation page). but this is how it should look once those are in place!

Reply:
Thanks @ulmononian . I'll wait until ecflow, mysql and spack-stack/1.4.1 are installed in the role-epic spaces. Then we can revisit!

Reply:
@zach1221 ecflow & mysql have been installed. the updated modulefile path is `/work/noaa/epic/role-epic/spack-stack/hercules/modulefiles`.

Reply:
updated